testing/bdd/features/handover/handover.feature

```diff
@@ -13,44 +13,66 @@ Feature: Handover algorithm
     Given an xml file <xml_file>
     And a handover node
     When it has sequence identifier <seq_id_1>
+    And it has authorsGroupIdentifier <group_1>
     And it has authorsGroupControlToken <token_1>
     And the document is generated
     And the document is processed 
-    # ^^^ ASSUMING THIS STEP COVERS CACHED TOKENS, IE THE SECOND DOCUMENTS DOESN'T HAVE TO FOLLOW IMMEDIATELY 
     And another document arrives 
     And it has sequence identifier <seq_id_2>
+    And it has authorsGroupIdentifier <group_1>
     And it has authorsGroupControlToken <token_2>
     And the document is generated
     And the document is processed
     Then the emitted document has <authorsGroupSelectedSequenceIdentifier> 
 
     Examples:
-    | seq_id_1   | token_1         | seq_id_2   | token_2         | authorsGroupSelectedSequenceIdentifier |  
-    | sequence_a | 12345           | sequence_b | 12346           | sequence_b                             |  
-    | sequence_a | 1               | sequence_b | 0               | sequence_a                             |  
-    | sequence_a | 0               | sequence_b |                 | sequence_a                             |  
-    | sequence_a | 999999999999999 | sequence_b | 999999999999998 | sequence_a                             |  
+    | seq_id_1   | group_1 | token_1   | seq_id_2   | group_2 | token_2   | authorsGroupSelectedSequenceIdentifier |  
+    | sequence_a | 1       | 12345     | sequence_b | 1       | 12345     | sequence_b                             |  
```

Comment:
I think the selected sequence here should be `sequence_b` since the token is the same as the first document, not greater.